spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/security/servlet/EndpointRequest.java

```diff
@@ -191,38 +194,53 @@ protected RequestMatcherProvider getRequestMatcherProvider(WebApplicationContext
 
 		private final boolean includeLinks;
 
+		@Nullable
```

Comment:
Re: `@Nullable`... You will notice I did **not** use it in the reactive counterpart - on purpose. I wanted to see what the recommendation from the team is in this area. I do not see it widely used in the SB codebase other than mostly on a few params to `@Endpoint` methods (selectors, params).  

Whatever direction we land I will make both imperative and reactive impls consistent.

Reply:
We currently don't use them in our own codebase so we may as well strip them out for now. We've got #10712 open to add them consistently.

Reply:
Will you handle that in your polish commit @philwebb ? I don't mind doing it - just let me know.